Uplift/Controllers/FilterViewController.swift

```diff
@@ -467,8 +467,8 @@ class FilterViewController: UIViewController, RangeSeekSliderDelegate {
         var modifiedIndices: [IndexPath] = []
 
         if classTypeDropdownData.dropStatus == .half || classTypeDropdownData.dropStatus == .down {
-            (classTypeDropdown.headerView(forSection: 0) as! DropdownHeaderView).downArrow.image = .none
-            (classTypeDropdown.headerView(forSection: 0) as! DropdownHeaderView).rightArrow.image = UIImage(named: ImageNames.rightArrow)
+            (classTypeDropdown.headerView(forSection: 0) as! OldDropdownHeaderView).downArrow.image = .none
+            (classTypeDropdown.headerView(forSection: 0) as! OldDropdownHeaderView).rightArrow.image = UIImage(named: ImageNames.rightArrow)
```

Comment:
can we unwrap this using an `if let` instead of doing this force cast

Reply:
Going off of my previous comment, this entire controller is going to be rewritten with the new Dropdown view once we're finished with facilities so there wouldn't be much point for now.